lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -2747,7 +2747,10 @@ protected void onPopulateNodeForVirtualView(
       if (values.size() > 1) {
         contentDescription.append(startOrEndDescription(virtualViewId));
         contentDescription.append(slider.formatValue(value));
+      } else if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.O){
```

Comment:
Let's remove this condition for now. I'll follow up with our a11y team for the compatibility support we need here. By removing, now it will announce "range start, 2" for range slider, and "value, 2" for regular slider for all talk back versions.

Reply:
I'll approve this PR and remove this condition internally.